build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md

```diff
@@ -2,12 +2,21 @@
 $category@: presentation
```

Comment:
Can we consolidate this with the older version of the generator?

Reply:
See latest commit